src/Fixer/Alias/RandomApiMigrationFixer.php

```diff
@@ -111,6 +111,19 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 $currIndex = $openParenthesis;
 
                 $tokens[$functionName] = new Token(array(T_STRING, $functionReplacement['alternativeName']));
+
+                if (0 === $count && 'random_int' === $functionReplacement['alternativeName']) {
+                    $tokens->insertAt(++$currIndex, array(
+                        new Token(array(T_LNUMBER, '0')),
+                        new Token(','),
+                        new Token(array(T_WHITESPACE, ' ')),
+                        new Token(array(T_STRING, 'getrandmax')),
+                        new Token('('),
+                        new Token(')'),
+                    ));
+
+                    $currIndex += 5;
```

Comment:
we are adding 6 tokens but moving the index by 5. ah, as we `++` it first. tricky